config.xhgui.yaml

```
@@ -10,14 +10,9 @@ hooks:
       fi
     service: db
   - exec: |
-      cat <<-EOF > .ddev/xhprof/xhprof_prepend.php
-      <?php
-      // DDEV's built in xhprof handler breaks our own.
-      // We'll temporarily override it here, but return control back later.
-      // If you don't want this behavior, comment out the hooks in ".ddev/config.xhgui.yaml".
-      return;
-      EOF
-  pre-stop:
+      cp .ddev/xhgui_prepend.php .ddev/xhprof/xhprof_prepend.php
```

This should be:

```suggestion
      cp /var/www/html/.ddev/xhgui_prepend.php /var/www/html/.ddev/xhprof/xhprof_prepend.php
```

I'm testing it on a project with a custom `working_dir`, where it didn't work.